VERSION_FAKE

```diff
@@ -1 +1 @@
-0.84.51
\ No newline at end of file
+0.86.0
```

Comment:
Are we burning the 86 release line like we did with 84?

Reply:
Yep.